pages/faq/troubleshooting/faq.md

```diff
@@ -26,25 +25,31 @@ title: FAQs
 * [What does it mean when a device type is discontinued?](#what-does-it-mean-when-a-device-type-is-discontinued-)
 
 ##### Can I use multiple containers?
-We are planning, and committed, to adding support for having multiple apps/containers running on a device. While the work towards multiple apps is in progress, as an interim solution, we do however  have a few users running [multiple containers within an app via docker-compose](https://resin.io/blog/multi-container-with-docker-compose-on-resin-io/) and [have done work with kubernetes](https://resin.io/blog/our-first-experiments-with-multi-container-apps/) in the same fashion.
+Multiple container applications are supported, beginning with resinOS v2.12.0. To run multiple containers, you will need to create a [Starter or Microservices type application][app-types] and include a `docker-compose.yml` file at the root of your project. You can reference the [multicontainer documentation][multicontainer] for more details on the supported configurations.
```

Comment:
We could say "... need to create or upgrade to a Starter or Microservices type..."